CONTRIBUTORS.md

```diff
@@ -189,8 +189,10 @@ Add your **i) Github username** and **ii) GitHub profile link** to the **Contrib
 179. [rbjrishabh22](https://github.com/rbjrishabh22)
 180. [dttrue](https://github.com/dttrue)
 181. [mrrmartin01](https://github.com/mrrmartin01)
-182. [Yash8452](https://github.com/Yash8452)
-183. [RainbowGenie](https://github.com/RainbowGenie)
-184. [Piyush](https://github.com/Piyush-linux)
-185. [Kosa12](https://github.com/kosa12)
-186. [HarshDekate](https://github.com/HarshDekate)
+182. [0xcryptotalent](https://github.com/0xcryptotalent)
```

Comment:
Please remove duplicate entry and follow the proper numbering order.

Closing this PR